newa/__init__.py

```diff
@@ -301,6 +302,33 @@ class EventType(Enum):
     COMPOSE = 'compose'
 
 
+class SysArchType(Enum):
```

Comment:
I'd call it just `Arch`, because architecture is a very much established concept, and there's hardly to be "non-sys" arch.

Reply:
Done.